zilliqa/src/message.rs

```diff
@@ -558,6 +558,7 @@ pub struct BlockHeader {
     pub timestamp: SystemTime,
     pub gas_used: EvmGas,
     pub gas_limit: EvmGas,
+    pub base_fee_per_gas: u128,
```

Comment:
Can we keep this as a fixed constant (the same value as `consensus_config.gas_price`). If we're adding it to the block header we will have a backwards compatibility issue, since we're changing the message format.